static/app/components/events/eventStatisticalDetector/aggregateSpanDiff.tsx

```diff
@@ -85,17 +93,91 @@ function AggregateSpanDiff({event, project}: AggregateSpanDiffProps) {
     relativeDays: 7,
     retentionDays: 30,
   });
-  const {data, isLoading, isError} = useFetchAdvancedAnalysis({
+
+  const {
+    data: rcaData,
+    isLoading: isRcaLoading,
+    isError: isRcaError,
+  } = useFetchAdvancedAnalysis({
     transaction,
     start: (start as Date).toISOString(),
     end: (end as Date).toISOString(),
     breakpoint: breakpointTimestamp,
     projectId: project.id,
+    enabled: !isSpansOnly,
+  });
+
+  const search = new MutableSearch('has:span.group');
```

Comment:
A comment here explaining why this condition is needed would be helpful to provide context